src/server/ApiRoutes/Hold.js

```diff
@@ -413,6 +413,10 @@ function createHoldRequestServer(req, res, pickedUpBibId = '', pickedUpItemId =
     return res.redirect(`${appConfig.baseUrl}/someErrorPage`);
   }
 
+  if (pickupLocation === 'edd'){
```

Comment:
That's fine. It works and it carries over the data for the bib/item. Just add a space after the closing parenthesis.